styles.css

```diff
@@ -297,3 +307,310 @@ footer {
+  .contact-form {
+    background: url("./images/desktop/image-geometry_6.svg") no-repeat,
+      url("./images/desktop/image-geometry_7.svg") no-repeat 100% 100%;
+    background-size: 170px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .contact-form-container {
+    padding: 0;
+    width: 542px;
+  }
+
+  form {
+    display: inline;
+    margin-top: 42px 0;
+  }
+
+  input {
+    width: 48%;
+    height: 60px;
+    margin-bottom: 42px;
+  }
+
+  #name {
+    float: left;
+  }
+
+  #email {
+    float: right;
+  }
+
+  textarea {
+    padding: 16px;
+    width: 100%;
+    height: 140px;
+  }
+
+  .navy-button {
+    width: 254px;
+  }
+
+  .contact-button,
+  .get-resume {
+    display: block;
+    margin: 30px auto;
+    margin-left: 133px;
+  }
+
+  .footer {
+    padding: 0 60px;
+    height: 8vh;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .social-networks {
+    gap: 32px;
+  }
+
+  .p6,
+  .pb3,
+  .pb4,
+  .pb2 {
+    display: none;
+  }
+
+  .p1 {
+    width: 98%;
+    height: 600px;
+  }
+
+  .p5 {
+    position: relative;
+    width: 90%;
+    height: 100px;
+    top: 50px;
+  }
+
+  .pi5 {
+    position: relative;
+    width: 92%;
+    left: 68px;
+  }
+
+  h1,
+  h2 {
+    font-size: 80px;
+  }
+
+  .project-heading h2 {
+    display: inline-block;
+    transform: rotate(-90deg) translateX(-50%);
+    padding-right: 50px;
+    font-weight: 400;
+    font-size: 110px;
+    line-height: 100%;
+    color: #1a2236;
+  }
+
+  /* EDITING */
+  .projects-container {
+    background: url("./images/desktop/image_dots.svg") no-repeat 30% 8%,
+      url("images/desktop/image-geometry_5.svg") no-repeat 0 90%,
+      url("images/desktop/image-geometry_2.svg") no-repeat 48% 52%,
+      url("images/desktop/image-geometry_3.svg") no-repeat 80% 82%,
+      url("images/desktop/image-geometry_4.svg") no-repeat 45% 90%;
+    background-size: 7%, 15%, 26%, 35%, 7%;
+  }
+
+  .project-wrapper {
+    display: grid;
+    width: 100%;
+    background: none;
+    grid-template-areas:
+      ". . . p1 p1 p1 p1 p1"
+      ". . . p1 p1 p1 p1 p1"
+      "p2 p2 . . . . p5 p5"
+      "p3 p3 p3 p3 p3 . p5 p5"
+      "p3 p3 p3 p3 p3 . . ."
+      ". . . . . p4 p4 .";
+  }
+
+  .p1 {
+    grid-area: p1;
+  }
+
+  .p2 {
+    grid-area: p2;
+  }
+
+  .p3 {
+    grid-area: p3;
+  }
+
+  .p4 {
+    grid-area: p4;
+  }
+
+  .p6 {
+    grid-area: p6;
+  }
+
+  .p5 {
+    display: flex;
+    flex-direction: column;
+    height: 10%;
+    grid-area: p5;
+  }
+
+  .pb5 {
+    position: relative;
+    top: 45px;
+    width: 73%;
+    left: 49px;
+    order: -1;
+  }
+
+  .p6,
+  .pb3,
+  .pb4,
+  .pb2 {
+    display: none;
+  }
+
+  a.button {
+    font-size: 0.8rem;
+  }
+
+  .project-view {
+    gap: 0;
+    justify-content: space-between;
+    background-color: #1a2236;
+    color: #fff;
+  }
+
+  .project-info {
+    padding: 50px;
+    height: 120px;
+    margin: 0 auto;
+  }
```

Comment:
Great job @Suigeneriz 🤝

- [ ] You may please fix this section and try to make your design a replica of the Figma template. You may as well make your design responsive as it reduces to a smaller screen. Most of the elements seem to be overlapping others as the screen size reduces
- Your design
![prject yours](https://user-images.githubusercontent.com/63917725/178114091-0ebe6873-d8e6-45e3-adf5-72010835f24e.PNG)

- Figma design
![project here Figma](https://user-images.githubusercontent.com/63917725/178114116-2e4ca75d-fd65-4e63-aec7-11b76181d4b3.PNG)